docs/release_process.md

```diff
@@ -0,0 +1,82 @@
+# Release Process
+
+So you want to release a new version, great!
+
+Here are the steps we follow.
+
+First, make sure all the desired pull requests have been merged.
+
+## Update the translations
+
+**TODO**
+
+## Get a clean environment
+
+Create a new virtual environment (`pew` makes this trivial), then get the code:
+
+    $ pew mktmpenv -p python3
+    $ git clone https://github.com/ideascube/ideascube
```

Comment:
Git push over http is available. It's even easier as git:// is sometime blocked by proxy/firewall depending of your connection.